src/index.js

```diff
@@ -6,73 +6,73 @@ const dependencies = new Map();
 const changedParents = new Set();
 const SOURCEMAPPING_URL = 'sourceMappingURL';
 const WAIT = 25;
-const touchParents = debounce(function () {
-	const now = Date.now(); // gives better performance then new Date()
-	for (let idx = 0, lst = changedParents.values(); idx < lst.length; idx += 1) {
-		fs.utimes(parent, now, now);
-	}
-	changedParents.clear();
+const touchParents = debounce(() => {
+    const now = Date.now(); // gives better performance then new Date()
+    for (let idx = 0, lst = changedParents.values(); idx < lst.length; idx += 1) {
+        fs.utimes(parent, now, now);
```

Comment:
Still on mobile... well. Not my code this one and is buggy. Set () and map () also give issues in some safari browsers and IE9.  Need refactoring here. See my comment on first issue